python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/asset_decorator.py

```diff
@@ -140,22 +43,25 @@ def sling_assets(
         partitions_def: Optional[PartitionsDefinition]: The partitions definition for this asset.
         backfill_policy: Optional[BackfillPolicy]: The backfill policy for this asset.
         op_tags: Optional[Mapping[str, Any]]: The tags for this asset.
-        group_name: Optional[str]: The group name for this asset.
-        code_version: Optional[str]: The code version for this asset.
-        freshness_policy: Optional[FreshnessPolicy]: The freshness policy for this asset.
-        auto_materialize_policy: Optional[AutoMaterializePolicy]: The auto materialize policy for this asset.
 
     Examples:
         # TODO: Add examples once API is complete
     """
     replication_config = validate_replication(replication_config)
     streams = get_streams_from_replication(replication_config)
 
+    group_name = dagster_sling_translator.get_group_name(replication_config)
+    code_version = hashlib.md5(str(replication_config).encode()).hexdigest()
```

Comment:
I could be wrong, but I _think_ you might have to add `usedforsecurity=False`. Government Dagster deployments require FIPS compliance and one of the things required for that is ack'ing that md5 usages aren't for security.

Reply:
oh nice! wasnt aware of `usedforsecurity=False`